TGC/events/Filibuster.txt

```diff
@@ -0,0 +1,623 @@
+### Filabusters ###
+
+	## William Walker invades Nicaragua
+		country_event = {
+			id = 8870780
+			title = "EVTNAME8870780"
+			desc = "EVTDESC8870780"
+			picture = "Cubanrebels"
+
+			fire_only_once = yes
+			trigger = {
+				tag = NIC
+				year = 1854
+				NOT = { year = 1860 }
+			}
+
+			mean_time_to_happen = {
+				days = 286
+			}
+
+			option = {
+				name = "EVTOPTA8870780"
+
+				prestige = -5
+				NIC = {
+					any_owned = {
+						limit = { province_id = 2198 }
+						add_core = FIL
+						secede_province = FIL
+					}
+					any_owned = {
+						limit = { province_id = 2197 }
+						add_core = FIL
+						secede_province = FIL
+					}
+					#2197 = {add_core = FIL}
+					2199 = {add_core = FIL}
+					#capital = 2199	
+				}
+
+				FIL = {
+					activate_technology = military_staff_system
+					activate_technology = iron_muzzle_loaded_artillery
+
+				}
+
+
+				188 = {
+					soldiers = {
+						#limit = {
+						#	has_pop_culture = dixie
+						#}
+						move_pop = 2198
+					}
+					officers = {
+						#limit = {
+						#	has_pop_culture = dixie
+						#}
+						move_pop = 2198
+					}
+				}
+
+				FIL = {
+					define_general = {
+						name = "William Walker"
+						personality = adventurer
+						background = relentless
+					}
+					money = 10000
+					[firearms] = 69
+				}
+
+				war = {
+					target = FIL
+					attacker_goal = { casus_belli = acquire_all_cores country = NIC }
+					defender_goal = { casus_belli = acquire_all_cores country = FIL }
+					call_ally = no
+				}
+
+				ai_chance = { factor = 1 }
+			}
+
+			option = {
+				name = "EVTOPTB8870780"
+
+				prestige = -5
+				NIC = {
+					any_owned = {
```

Comment:
When having only one province possible like this use random_owned as it has the same effect but is more neatly displayed in game.